Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.ts

```diff
@@ -98,24 +98,25 @@ export class UserDetailComponent implements OnInit, OnDestroy {
   public openChangePopup(): void {
     const wrapper = document.getElementsByClassName('pb-margin-20')[0];
     wrapper.classList.add('blur');
-    const dialogConfig = new MatDialogConfig();
-    dialogConfig.disableClose = true;
-    dialogConfig.autoFocus = true;
-    dialogConfig.height = '25vh';
-    dialogConfig.width = '30vw';
     const name = {
       firstNames: this.person.Firstname,
       lastNames: this.person.Surname,
       Titel: this.person.Title,
       Id: this.person.Id
     };
-    dialogConfig.data = {
-      Id: name.Id,
-      Firstname: name.firstNames,
-      Lastname: name.lastNames,
-      Titel: name.Titel
+    const dialogConfig : MatDialogConfig = {
+      disableClose : true,
```

Comment:
Why should we hinder the user from clicking elsewhere to close?

Reply:
I've implement a button to close the Dialog :) 

Or do you just want the Dialog to pop up and close without button ?

Reply:
Just do both :)